src/flows/components/panel/FlowPanel.tsx

```diff
@@ -206,7 +148,7 @@ const FlowPanel: FC<Props> = ({
                 <FeatureFlag name="flow-debug-queries">
                   <SquareButton
                     icon={IconFont.BookCode}
-                    onClick={printMap}
+                    onClick={() => printMap(id, true)}
```

Comment:
why wouldn't `printMap` just be handling true cases?

Reply:
in case you wanna test it with the actual notebook state